### DISCUSSION TITLE
IOS volume slider placement

### DISCUSSION THREAD

**Discussion by tottoszsolt** [KRTirtho/spotube#1699] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The problem is with the placement of the volume slider, in the app.. On my device (Iphone 14) the UI looks like this: 
![448240686_498705262593442_6840107987900200054_n](https://github.com/KRTirtho/spotube/assets/43206967/ccbf1f17-faa4-430c-b349-2b12c005a7bf)
As you can see the volume slider is really really close to the bottom panel, so when I swipe my volume, it will switch apps because of my iphone's default gestures

### Expected Behavior

We only need to move up it like to be right below the lqueue and Lyrics button

### Steps to reproduce

1. Open the app on any Iphone
2. See the volume slider yourself

### Operating System

IOS

### Spotube version

3.7.1

### Installation source

GitHub Releases (Binary)

### Self grab

- [X] I'm ready to work on this issue!

**Comment by KRTirtho** [comment, marked as answer]
You can simply scroll down. There's more space/gap in the bottom

**Reply by tottoszsolt** [reply]
Thank you! Indeed, I didnt saw it earlier.
